apps/frontend/src/pages/projects/components/project-create-form/project-create-form.tsx

```diff
@@ -13,12 +19,19 @@ type Properties = {
 };
 
 const ProjectCreateForm = ({ onSubmit }: Properties): JSX.Element => {
-	const { control, errors, handleSubmit } = useAppForm<ProjectCreateRequestDto>(
-		{
+	const { control, errors, handleSubmit, handleTrigger } =
+		useAppForm<ProjectCreateRequestDto>({
 			defaultValues: DEFAULT_PROJECT_CREATE_PAYLOAD,
 			validationSchema: projectCreateValidationSchema,
-		},
-	);
+		});
+
+	const descriptionValue = useFormWatch({
+		control,
+		defaultValue: "",
+		name: "description",
+	});
+
+	const isDescriptionCounterShown = Boolean(!errors["description"]?.message);
```

Comment:
! is enough here, let's remove Boolean wrapper
```suggestion
	const isDescriptionCounterShown = !errors["description"]?.message;
```